spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -479,7 +479,7 @@ private void setupConfForPySpark(SparkConf conf) {
 
     //Only one of py4j-0.9-src.zip and py4j-0.8.2.1-src.zip should exist
     String[] pythonLibs = new String[]{"pyspark.zip", "py4j-0.9-src.zip", "py4j-0.8.2.1-src.zip",
-      "py4j-0.10.1-src.zip"};
+      "py4j-0.10.3-src.zip"};
```

Comment:
Shouldn't "py4j-0.10.1-src.zip" need to be in the array for in case of spark 2.0.0?